contracts/libraries/Math.sol

```diff
@@ -4,12 +4,29 @@ pragma solidity ^0.8.0;
 // Libraries
 import {Errors} from "@libraries/Errors.sol";
 
+// Custom types
```

Comment:
We don't do line breaks between library imports, so can you delete the one above?